docs/monitors/statsd.md

```diff
@@ -15,16 +15,15 @@ datapoints.  It listens on a configured address and port in order to
 receive the statsd metrics.  Note that this monitor does not support statsd
 extensions such as tags.
 
-Supports Spark 2.2.0 and later.
-
 The monitor supports the `Counter`, `Timer`, `Gauge` and `Set` types which
 are dispatched as the SignalFx types `counter`, `gauge`, `gauge` and
 `gauge` respectively.
 
 **Note that datapoints will get a `host` dimension of the current host that
-the agent is running on, not the host from which the statsd metric was
-sent.  For this reason, it is recommended to send statsd metrics to a local
-agent instance.**
+the agent is running on, not the host from which the statsd metric was sent.
+For this reason, it is recommended to send statsd metrics to a local agent
+instance. If you don't want the `host` dimension, you can set
+`disableHostDimensions: true` on the monitor configuration**
```

Comment:
Could you please update the example in the _Formatting metric name_ section: `metric: "{traffic}.{action}"` -> `metricName: "{traffic}.{action}"`

Reply:
Fixed.